### ISSUE TITLE
Clicking on empty space behavior is unexpected in 0.217.0-5

### ISSUE THREAD

**Issue by steelbrain** [atom/tree-view#1075]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Clicking on empty space about tree view items has two different behaviors:

- If it's the active item, it's toggled (for dirs)
- If it's text width is equal to or less than the name of longest time, it's selected and toggled as expected
- If it's text width is greater than the longest name and isn't the active item, clicking on empty space in front of it has **zero** effect

### Steps to Reproduce

1. Open any repository
2. Click on the empty space in front of the name of a dir/file that's not active and is the longest

**Expected behavior:** [What you expect to happen]

The item is selected or toggled regardless

**Actual behavior:** [What actually happens]

Mentioned above

**Reproduces how often:** [What percentage of the time does it reproduce?]

Everytime

### Versions

```
Atom    : 1.17.0-beta3
Electron: 1.3.14
Chrome  : 52.0.2743.82
Node    : 6.5.0
```

Tree-View v0.217.0-5

### Additional Information

I'm on macOS if it matters, also here's a gif

![kapture 2017-04-20 at 22 54 09](https://cloud.githubusercontent.com/assets/4278113/25245012/5c349b9e-261c-11e7-8d14-fcacfc39622d.gif)

**Comment by rsese** [comment]
I can reproduce - with the Atom Dark theme, I even see a faint guide that shows the cutoff that you mention:

![select-cutoff](https://github-slack.s3.amazonaws.com/monosnap/Project__sandboxhello-world_2017-04-20_17-35-15.png)

The behavior is different in 1.16, it behaves like your described expected behavior so I'll go ahead and label it as a regression for now.

**Comment by maxbrunsfeld** [comment]
This will be fixed in the next 1.17 beta release.
